packages/mdc-ripple/_mixins.scss

```diff
@@ -107,7 +109,9 @@
 
       content: "";
     }
-  } @else {
+  }
+
+  @else {
```

Comment:
Is this really our standard practice for `@else` blocks (on their own line with an empty line in between), and was this performed by our fix task when updating for this PR or is it unrelated?

Tangentially, it appears we have 3 different styles of `@else` - cuddled, on its own line with no empty line in between, and this. Personally I find the empty line a bit confusing since the `@else` is conceptually connected to what's above it and empty lines usually serve the purpose of disconnecting from what's above.

Reply:
I personally don't love this style. Is there a style guide you can point to that recommends it?

Reply:
Agreed, this is a weird format change. I think it snuck it here with an auto-fix.

I think I need to make another PR that lints our @if and @else (a la https://github.com/kristerkari/stylelint-scss/blob/master/docs/examples/if-else.md). Then we can cuddle our @elses across the code base